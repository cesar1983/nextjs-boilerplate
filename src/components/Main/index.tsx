const Main = () => (
  <main>
    <h2>Trad</h2>
  </main>
)

export default Main
