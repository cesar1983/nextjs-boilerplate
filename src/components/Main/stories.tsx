import { Story, Meta } from '@storybook/react/types-6-0'
import Main from '.'

export default {
  title: 'Main',
  component: Main
} as Meta

export const Component1: Story = (args) => <Main {...args} />

export const Component2: Story = (args) => <Main {...args} />

Component2.args = {
  title: 'Title Default',
  description: 'description default'
}
