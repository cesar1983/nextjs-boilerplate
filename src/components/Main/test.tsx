import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should render the trad heading', () => {
    const { container } = render(<Main />)

    expect(screen.getByRole('heading', { name: /trad/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
