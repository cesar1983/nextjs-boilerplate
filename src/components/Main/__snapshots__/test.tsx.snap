// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the trad heading 1`] = `
<main>
  <h2>
    Trad
  </h2>
</main>
`;
